src/js/controller/model.js

```diff
@@ -93,8 +94,11 @@ define([
                     return;
 
                 case events.JWPLAYER_MEDIA_TYPE:
-                    this.mediaModel.set('mediaType', data.mediaType);
-                    break;
+                    if (this.mediaModel.get('mediaType') !== data.mediaType) {
+                        this.mediaModel.set('mediaType', data.mediaType);
+                        this.mediaController.trigger(type, evt);
```

Comment:
Can we have this as a break instead and rely on the ```this.mediaController.trigger(type, evt);``` at the end of the handler instead?